test/e2e/storage/testsuites/volume_stress.go

```diff
@@ -187,7 +185,6 @@ func (t *volumeStressTestSuite) DefineTests(driver storageframework.TestDriver,
 		}
 		wg.Wait()
 
-		errs = append(errs, storageutils.TryFunc(l.driverCleanup))
```

Comment:
Now that half of cleanup is done in `defer` and second half in `ginkgo.DeferCleanup`, can they be executed in an unexpected order? This code would call `driverCleanup` and then rest of `defer` of a test and then `ginkgo.DeferCleanup`. With this PR,  the order is different. I don't think it matters at *this* particular line, but reviewing all other usages would be very cumbersome.

Reply:
The order is deterministic: first defer inside the function, then `ginkgo.DeferCleanup`, in first-in-first-out order.

This is what one would normally expect: things that get set up first also get torn down last and thus remain usable while code runs that depends on them. The only time where I had problems with that was when some later code depended on earlier code cancelling a context during shutdown (or something like that). But this manifested as an obvious test failure and thus showed up. I don't remember where exactly that was.

> There are lot of cleanups that might have been done by ginkgo.DeferCleanup instead of defer.

I am doing more of those in https://github.com/kubernetes/kubernetes/pull/112923, but it's already a too big PR. Once all of my changes are in, it would make sense for some other volunteer to spend more time on further cleanups.